Gems/ScriptEvents/Code/Source/ScriptEventMethod.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ScriptEvents/ScriptEventMethod.h"
+
+#include <AzCore/RTTI/BehaviorContext.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/std/string/regex.h>
+
+namespace ScriptEvents
+{
+    void Method::FromScript(AZ::ScriptDataContext& dc)
+    {
+        if (dc.GetNumArguments() > 0)
+        {
+            AZStd::string name;
+            if (dc.IsString(0) && dc.ReadArg(0, name))
+            {
+                m_name.Set(name.c_str());
+            }
+
+            if (dc.GetNumArguments() > 1)
+            {
+                AZ::Uuid returnType;
+                if (dc.ReadArg(1, returnType))
+                {
+                    m_returnType.Set(returnType);
+                }
+            }
+        }
```

Comment:
This is another random change

Reply:
@lsemp3d @carlitosan Was this file removed and is this PR trying to re-add it back?